api/pom.xml

```diff
@@ -47,7 +47,17 @@
 			<version>0.3m</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>1.7.5</version>
```

Comment:
Because you have these in your parent pom's dependencyManagement section you shouldn't specify version here and instead let it inherit that from whatever you set in the root level pom

Reply:
Good to know, thanks
PR#28 should fix this if you can take a look at that